test_quickjs.py

```diff
@@ -354,6 +422,17 @@ def test_deep_recursion(self):
         f.set_max_stack_size(2000 * limit)
         self.assertEqual(f(limit), limit)
 
+    def test_add_callable(self):
+        f = quickjs.Function(
+            "f", """
+            function f() {
+                return pfunc();
+            }
+        """)
+        f.add_callable("pfunc", lambda: 42)
+
+        self.assertEqual(f(), 42)
+
```

Comment:
Need a test for re-entrancy

Reply:
The commit that removed the restriction (654eb7b587c9f2350192f61de857c8ab4a1a7667) also had a few test changes. Did you have anything specific in mind?

Reply:
Ah, I missed those. Still, would be worth having a test with a time limit, just to check that it doesn't crash, and maybe worth adding a Python -> JS -> Python -> JS -> Python test to properly test JS -> Python reentrance.

Reply:
Disallowed for now with a test that expects an exception.